src/content/api/loaders.md

```diff
@@ -436,6 +436,13 @@ Emit a file. This is webpack-specific.
 Access to the `compilation`'s `inputFileSystem` property.
 
 
+### `this.mode`
+
+The [`mode` configuration](/configuration/mode/) for webpack.
+
+Example values: `"production"`, `"development"`, or `"none"`
```

Comment:
We are trying to use single quotes for the strings consistently:

```suggestion
Possible values: `'production'`, `'development'`, `'none'`
```

Reply:
I was going for consistency with the existing examples for [`this.resource`](https://github.com/webpack/webpack.js.org/blob/73be8eff00533125288af81d9c731bd5403d7a12/src/content/api/loaders.md#thisresource), [`this.resourceQuery`](https://github.com/webpack/webpack.js.org/blob/73be8eff00533125288af81d9c731bd5403d7a12/src/content/api/loaders.md#thisresourcequery), and [`this.target`](https://github.com/webpack/webpack.js.org/blob/73be8eff00533125288af81d9c731bd5403d7a12/src/content/api/loaders.md#thistarget) in this same doc, but I'll update to use single quotes if that is more consistent with the future.